client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -69,6 +72,11 @@ const hosting: Flow = {
 
 					setPlanCartItem( {
 						product_slug: productSlug,
+						extra: {
+							...( queryParams?.utm_source && {
+								hideProductVariants: queryParams.utm_source === 'wordcamp',
```

Comment:
**Note:**

We'll want to make this "wordcamp" utm_source a constant somewhere 🤔

Reply:
Considering this is a temporary change, I think we should be okay with this. We can revert this change once the promo is over.